src/ol/webgl/Helper.js

```diff
@@ -264,7 +264,7 @@ function releaseCanvas(key) {
  * ### Specifying attributes
  *
  *   The GPU only receives the data as arrays of numbers. These numbers must be handled differently depending on what it describes (position, texture coordinate...).
- *   Attributes are used to specify these uses. Use {@link enableAttributeArray_} and either
+ *   Attributes are used to specify these uses. Use {@link module:ol/webgl/Helper~WebGLHelper#enableAttributes endableAttributes} and either
  *   the default attribute names in {@link module:ol/webgl/Helper.DefaultAttrib} or custom ones.
```

Comment:
`ol/webgl/Helper.DefaultAttrib` was removed in #9994. What should the description be here?

Reply:
I'd suggest
```
 *   Attributes are used to specify these uses. Specify the attribute names with
 *   {@link module:ol/webgl/Helper~WebGLHelper#enableAttributes enableAttributes()} (see code snippet below).
```